_playwright-tests/UI/helpers/navHelpers.ts

```diff
@@ -34,12 +34,10 @@ export const navigateToRepositories = async (page: Page) => {
 const navigateToTemplatesFunc = async (page: Page) => {
   await page.goto('/insights/content/templates');
 
-  const repositoriesListPage = page.getByText(
-    'View all content templates within your organization.',
-  );
+  const templateText = page.getByText('View all content templates within your organization.');
 
   // Wait for either list page or zerostate
-  await repositoriesListPage.waitFor({ state: 'visible' });
+  await templateText.waitFor({ state: 'visible' });
```

Comment:
Hmm, is this change just to keep the variable shorter so it does not wrap the command? :D

Reply:
I think the name change was meant to better describe the selector.

Reply:
But I suppose your point was the commit wasn't clear.

Reply:
The commit only says "Parameter update". No commit message, so it is not clear why you did this change

Reply:
Hi, I agree that list of templates should not be called `repositoriesListPage`.
While `templateText` seems something specific to one template ( `templateListPage`?), I could not think of something better that would not be much longer.